src/components/Post.jsx

```diff
@@ -1,3 +1,13 @@
+import { useEffect, useState } from "react";
```

Comment:
why is there duplicate import of modules? please fix that and test if the application runs without issue.